opensearchpy/_async/client/__init__.py

```diff
@@ -1948,21 +1948,31 @@ async def get_script_languages(self, params=None, headers=None):
         )
 
     @query_params()
-    async def close_point_in_time(self, body=None, params=None, headers=None):
+    async def get_point_in_time(self, params=None, headers=None):
```

Comment:
Should we make it list_add instead of get? This will be in-sync with the opensearch backend code/api.

Reply:
Sure, update it thanks!